README.md

```diff
@@ -49,7 +52,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+belle = town.residents[1];
```

Comment:
You don't need semicolons in Ruby -- but if it's a conscious choice, go for it! Just know that the vast majority of Ruby developers don't use them.